homeassistant/components/light/demo.py

```diff
@@ -129,9 +137,14 @@ def turn_on(self, **kwargs):
         if ATTR_EFFECT in kwargs:
             self._effect = kwargs[ATTR_EFFECT]
 
+        # For non-polling lights it is necessary to schedule an update
+        # after making a change, here just to give an example.
```

Comment:
When an entity disabled polling, it is itself responsible for pushing state updates to Home Assistant.